docs/ovn_k8s.conf.5

```diff
@@ -72,6 +92,9 @@ The Kubernetes API authentication token (not required if kubeconfig is given).
 .SH [OvnSouth]
 .TP
 \fBaddress\fR=
+This is the url used to access the southbound database server. The scheme may be ssl or tcp.
+When ssl is used, the certs must be provided. The host must be a host IP address, not name.
+The port, by default, is 6442. E.g., ssl://1.2.3.4:6642
 .TP
 \fBclient-privkey\fR=
```

Comment:
Same as before, but with ``sb`` instead of ``nb``.

Reply:
Thanks, changed.

Reply:
Same comments as above for the northbound.